src/app/components/charts/LineChart.tsx

```diff
@@ -33,7 +33,7 @@ const LineChartCmp = <T extends object>({
   tooltipActiveDotRadius = 5,
   withLabels,
 }: LineChartProps<T>): ReactElement => (
-  <ResponsiveContainer width="100%" aspect={4}>
+  <ResponsiveContainer width="100%">
```

Comment:
Avg Transaction Size is not rendering without aspect ratio

Reply:
Good point.